webvtt.go

```diff
@@ -203,6 +205,21 @@ func ReadFromWebVTT(i io.Reader) (o *Subtitles, err error) {
 	return
 }
 
+func parseTextWebVTT(i string) (o Line) {
+	match := voiceTagRegexp.FindStringSubmatch(i)
```

Comment:
you need to check that match is `!= nil` because if it's `nil` it means there are no matches